BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudPlayerState.cs

```diff
@@ -394,5 +394,150 @@ public BrainCloudPlayerState(BrainCloudClient client)
             ServerCall sc = new ServerCall(ServiceName.PlayerState, ServiceOperation.UpdateContactEmail, data, callback);
             _client.SendRequest(sc);
         }
+
+        /// <summary>
+        /// Clear the user's status.
+        /// </summary>
+        /// <remarks>
+        /// Service Name - PlayerState
+        /// Service Operation - ClearUserStatus
+        /// </remarks>
+        /// <param name="statusName">
+        /// The name of the status.
+        /// <param name="success">
+        /// The success callback.
+        /// </param>
+        /// <param name="failure">
+        /// The failure callback.
+        /// </param>
+        /// <param name="cbObject">
+        /// The user object sent to the callback.
+        /// </param>
+        public void ClearUserStatus(
+            string statusName,
+            SuccessCallback success = null,
+            FailureCallback failure = null,
+            object cbObject = null)
+        {
+            Dictionary<string, object> data = new Dictionary<string, object>();
+            data[OperationParam.PlayerStateServiceStatus.Value] = statusName;
+
+            ServerCallback callback = BrainCloudClient.CreateServerCallback(success, failure, cbObject);
+            ServerCall sc = new ServerCall(ServiceName.PlayerState, ServiceOperation.ClearUserStatus, data, callback);
+            _client.SendRequest(sc);
+        }
+
+        /// <summary>
+        /// Extends the Status.
+        /// </summary>
+        /// <remarks>
+        /// Service Name - PlayerState
+        /// Service Operation - ExtendUserStatus
+        /// </remarks>
+        /// <param name="statusName">
+        /// The name of the status.
+        /// <param name="additionalSecs">
+        /// The number of seconds to add.
+        /// <param name="details">
+        /// The details of the status.
+        /// <param name="success">
+        /// The success callback.
+        /// </param>
+        /// <param name="failure">
+        /// The failure callback.
+        /// </param>
+        /// <param name="cbObject">
+        /// The user object sent to the callback.
+        /// </param>
+        public void ExtendUserStatus(
```

Comment:
this call is wrong, You need to allow the user to pass in details. 
You should have 3 parameters:  statusName, additionalSecs, and details
Refer to how you implemented SubmitCrashReport to successfully convert the details string to a dictionary.